cartopy/docs/v0.19/.buildinfo

```diff
@@ -0,0 +1,4 @@
+# Sphinx build info version 1
```

Comment:
Please delete this file.

Reply:
Done, any others hanging around too? I just pulled straight from the CircleCI doc build.

Reply:
No, I think that's all.